addons/sale_margin/views/sale_order_views.xml

```diff
@@ -14,7 +14,7 @@
                         <field name="margin" class="oe_inline"/>
                         <field name="amount_untaxed" invisible="1"/>
                         <span class="oe_inline" invisible="amount_untaxed == 0">
-                            (<field name="margin_percent" nolabel="1" class="oe_inline" widget="percentage"/>)
+                            <field name="margin_percent_display_text" nolabel="1" class="oe_inline"/>
```

Comment:
So basically, if I understand correctly, you create a dedicated field so that if they hide the margin they also hide the parenthesis ?

If that's the only reason, it's not worth it and not something we want to fix IMO.